lib/findingaids/ead/behaviors.rb

```diff
@@ -64,13 +64,26 @@ def ead_date_display(parts = Array.new)
     return parts.join(" ")
   end
 
-  # Returns the language terms as string from a given three-letter code found in language_map.properties
-  def get_language_from_code code, properties = Hash.new
-    file = File.new(File.path(Rails.root + "config/translation_maps/language_map.properties"))
-    IO.foreach(file) do |line|
-      properties[$1.strip] = $2 if line =~ /([^=]*)=(.*)\/\/(.*)/ || line =~ /([^=]*)=(.*)/
-    end
-    properties[code].nil? ? nil : properties[code].strip
+  # Returns the language term as string from a given three-letter code found in the ISO 639 gem
+  def get_language_from_code code
+    iso_lang_code = ""
+    iso = ISO_639.search(code) 
+    #returns an array of arrays 
+    iso.each_index{|la|
+      iso[la].each{|l|
+      #if code is found in language array of arrays
+        if l == code
+          #the ISO English equivalent of the code or the language
+          #always in the 4th position of the array
+          iso_lang_code = iso[la][3]
+          #breaking loop if match found 
+          break
+        end
+      }
+           
+    }
+    iso_lang_code
+
```

Comment:
@barnabyalter,

Ah yes, I can do it by english_name here. I had copied and pasted this code from ichabod where I couldn't necessarily do that because I was unsure as to what format the language was coming in. But the finding aid language values are more controlled, so I'll change it. Thanks.